.circleci/config.yml

```diff
@@ -13,12 +13,18 @@ commands:
           command: |
             [ ! -d "/tmp/artifacts" ] && mkdir /tmp/artifacts &>/dev/null
 
-  set_verision_variable:
+  set_version_variable:
     description: "Set the VERSION environment variable"
     steps:
       - run:
           command: |
-            echo "export VERSION=$(grep 'Version:' /tmp/src/simple-social-icons.php | awk -F: '{print $2}' | sed 's/^\s//')" >> ${BASH_ENV}
+            PLUGIN_VERSION=$(grep 'Version:' /tmp/src/simple-social-icons.php | awk -F: '{print $2}' | sed 's/^\s//')
+            if [ ! $PLUGIN_VERSION ]
+              then
+              echo "Didn't find a version of the plugin"
+              exit 1
```

Comment:
Because we're now deleting a tag, like `tags/3.0.2`, this could delete all tags if `$VERSION` is `''`.

Reply:
This change to `set_version_variable` is just a helper, it's not the main fix in the PR